src/components/Lobby/Lobby.jsx

```diff
@@ -34,14 +34,14 @@ export class Lobby extends React.Component {
 
     render() {
         return (
-            <div className={styles.lobbyPage}>
+            <>
```

Comment:
is this nameless tag intentional?

Reply:
yeah, react complains when you return multiple tags, they need to be wrapped in something